app/assets/stylesheets/common/base/category-list.scss

```diff
@@ -43,9 +43,8 @@
 
 .category-boxes,
 .category-boxes-with-topics {
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: flex-start;
+  display: grid;
+  gap: 1.5em;
```

Comment:
Given the incoming release and the iOS 12 policy (see: /t/64060/25) maybe this PR should wait for the first 3.1 beta?

Reply:
Ah, true prefix-less `gap` was first available in 12.0! It was "`gap` for flexbox" that was introduced in 14.